packages/api-explorer/src/components/AuthForm/index.jsx

```diff
@@ -79,14 +84,8 @@ class AuthForm extends Component {
     onChange(mergeAuths)
   }
 
-  onSecuritySectionSubmit (e) {
-    const {onSubmit} = this.props
```

Comment:
This was broken, because this method was passed as argument below, and so the `this` was not refered to the class. So was not executed the `preventDefault`.
I moved the `preventDefault` closer to the form tag.